Documentation/operations/upgrade.rst

```diff
@@ -451,11 +451,20 @@ Helm Options
   gateway nodes. Once the connectivity is restored, clients will need to
   reconnect.
 
+Added Metrics
+~~~~~~~~~~~~~
+* ``cilium_operator_allocation_duration_seconds``
+* ``cilium_operator_release_duration_seconds``
+
 Removed Metrics/Labels
 ~~~~~~~~~~~~~~~~~~~~~~
 
 * ``cilium_operator_identity_gc_entries_total`` is removed. Please use ``cilium_operator_identity_gc_entries`` instead.
 * ``cilium_operator_identity_gc_runs_total`` is removed. Please use ``cilium_operator_identity_gc_runs`` instead.
+* ``cilium_operator_ipam_available`` is removed. Please use ``cilium_operator_ipam_available_interfaces`` instead.
+* ``cilium_operator_ipam_allocation_ops`` is removed. Please use ``cilium_operator_ipam_ip_allocation_ops`` instead.
+* ``cilium_operator_ipam_release_ops`` is removed. Please use ``cilium_operator_ipam_ip_release_ops`` instead.
+* The label of ``status`` in ``cilium_operator_ipam_interface_creation_ops`` is removed.
```

Comment:
nit: Can you please move this up into a new section for v1.13 upgrade notes? It doesn't make sense to state this in the v1.11 section. If there is not the existing `ADded metrics` titles etc. in the v1.13 section above, please create it.

Reply:
current no `1.13 Upgrade Notes` in [Documentation/operations/upgrade.rst](https://github.com/cilium/cilium/pull/20755/files/73a4efa0a2fb4eb2d60aba8e2d533955e38423f7#diff-76c7d3d61bd3c6afeb8e936896c7a01046d530840abf499fd118e88e75d1a504), i will add it.